StaticAnalyzer/views/android/static_analyzer.py

```diff
@@ -508,3 +524,32 @@ def gen_downloads(app_dir, md5, icon_path=''):
 
     except:
         PrintException("Generating Downloads")
+
+
+def get_app_name(app_path, app_dir, tools_dir, is_apk):
+    """ get app name """
+    
+    data = ""
+    if is_apk:
+        output_dir = os.path.join(app_dir, "apktool_out")
+        strings_file = os.path.join(output_dir, "res/values/strings.xml")
```

Comment:
Are you sure for all versions of APK
`(output_dir, "res/values/strings.xml")` this path exists?

If not `os.path.exists` needs to be done before attempting to read strings.xml in all case.

Reply:
I added a judgment to the back that did not find the string_file file.